### ISSUE TITLE
Suggestion for Deckboard

### ISSUE THREAD

**Issue by spesvictus** [rivafarabi/deckboard#303]
Integrate Auto Hotkey into Deckboard. This should extend Deckboard's capabilities a great deal.

**Comment by Yurlyn** [comment]
I don't think that will really work since AHK is a completely separate program and something has to be written for that to work with Deckboard. What would you like to do with it? Because I have a Run Program button and have set the executable file to the *.ahk file I need.
I have "#SingleInstance Ignore" and "#NoTrayIcon" at the top of my script so it doesn't show up in the system tray and the script closes itself after it's done (it just resizes and moves some of my windows in case my monitor disconnects). So with this it doesn't show itself anywhere but still does its job. this might be a solution for you too.
